phoenix-core/src/test/java/org/apache/phoenix/memory/MemoryManagerTest.java

```diff
@@ -19,19 +19,26 @@
 
 import static org.junit.Assert.assertTrue;
 import static org.junit.Assert.fail;
+import static org.mockito.Matchers.anyLong;
+import static org.mockito.Mockito.atLeastOnce;
+import static org.mockito.Mockito.spy;
 
+import com.google.common.base.Stopwatch;
```

Comment:
Don't use the guava StopWatch here. We have run into conflicts before because of mismatches in guava versions. You can use the PhoenixStopWatch class here instead.